### PR TITLE
Surfaces instance metadata id if present in footer

### PR DESCRIPTION
**Pull request by ajainarayanan**
Build: https://builds.cask.co/browse/CDAP-UDUT272

### CODE REVIEW

**Review by elfenheart — APPROVED** [review]
👍lgtm

File: cdap-ui/app/cdap/components/Footer/index.js [hunk 1]
```diff
@@ -16,19 +16,24 @@
...
 import React from 'react';
 import { Theme } from 'services/ThemeHelper';
-
+import If from '../If';
 require('./Footer.scss');
 
 export default function Footer() {
   const footerText = Theme.footerText;
   const footerUrl = Theme.footerLink;
+  // 'project-id-30-characters-name1/instance-id-30-characters-name';
```
Comment (anwar6953) [inline cdap-ui/app/cdap/components/Footer/index.js:25]: What is this comment?

Comment (ajainarayanan) [reply cdap-ui/app/cdap/components/Footer/index.js:25]: Sample format of how it looks with Max 30 characters

**Comment by ajainarayanan** [comment]
Latest build https://builds.cask.co/browse/CDAP-UDUT272-1

**Pull request merged**